ush/preamble.sh

```diff
@@ -0,0 +1,92 @@
+#! /usr/bin/env bash
```

Comment:
Please remove this from your branch. This won't go into the GFS with v16 and still needs sign off from NCO for v17.